java/ql/src/experimental/semmle/code/java/Logging.qll

```diff
@@ -0,0 +1,26 @@
+/**
+ * Provides classes and predicates for working with logger's and their components.
+ */
+
+import java
+
+/** Models a call to a logging method.. */
+class LoggingCall extends MethodAccess {
+  LoggingCall() {
+    exists(RefType t, Method m |
```

Comment:
Can you please include:
- [`java.util.logging.Logger`](https://docs.oracle.com/en/java/javase/15/docs/api/java.logging/java/util/logging/Logger.html)
- [`java.lang.System.Logger`](https://docs.oracle.com/en/java/javase/15/docs/api/java.base/java/lang/System.Logger.html) (added in Java 9)

Reply:
@Marcono1234 I am trying to add flogger here. I want to refrence [`LoggingApi` interface  ](https://javadoc.io/static/com.google.flogger/flogger/0.5.1/com/google/common/flogger/LoggingApi.html). I tried something like 
```codeql
hasQualifiedName("com.google.common.flogger", "LoggingApi") 

hasQualifiedName("com.google.common.flogger", "LoggingApi<API>") 

hasQualifiedName("com.google.common.flogger", "LoggingApi<>") 
```

However, all of these fail to detect a line similar to shown below.
```java
private static final FluentLogger LOGGER = FluentLogger.forEnclosingClass();
LOGGER.atInfo().log("sink");
```

Any idea what I may be missing here? 

CC: @owen-mc

Reply:
Might be related to https://github.com/github/codeql/pull/5099#discussion_r585892647, maybe one of these works when that is changed.
Though I am always uncertain when it comes to matching generic types as well.